tenbilac/committee.py

```diff
@@ -0,0 +1,189 @@
+"""
+This is Tenbilac!
+This is a net (Wnet in the future) ensemble class.
+"""
+
+import numpy as np
+from multiprocessing import Process, Pipe, Queue
+from itertools import izip  
+
+import logging
+logger = logging.getLogger(__name__)
+
+from . import train
+from . import utils
+
+
+def spawn(f):  
+	def fun(pipe,x):  
+		pipe.send(f(x))  
+		pipe.close()  
+	return fun  
+
+def parmap(f, X, ncpu):  
+	"""
+	This is an alternative to multiprocessing.Pool to avoid the limitations of the package (pickling stuff...)
+	
```

Comment:
OK -- as long as it works.... but I guess that there is a way to get this working with multiprocessing pool as well, maybe by redefining wrappers around the operations to be done in parallel, or by letting the workers load all their data from disk. But as I said, OK for me to play with this!